spec/features/editing_tags/edit_tags_blank_primary_org_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+RSpec.feature "Showing a publishing error when there is no Primary Organisation selected" do
+  let(:initial_blank) { { "content_id" => "", "internal_name" => "blank" } }
+  let(:organisation) { { "content_id" => SecureRandom.uuid, "internal_name" => "test_org" } }
```

Comment:
Convention in our feature tests is to not use let and we instead use instance variables (`@var`) for variables used in multiple scopes.

Reply:
This is an artefact of my lack of knowledge here - I based this code off another tag spec test, perhaps that needs to be updated too?

Reply:
Sure, which test?